cmd/juju/application/deploy.go

```diff
@@ -1702,16 +1702,6 @@ func (c *DeployCommand) validateCharmSeriesWithName(series, name string) error {
 func charmValidationError(charmSeries, name string, err error) error {
 	if err != nil {
 		if errors.IsNotSupported(err) {
-			// output the warning information to help track down the error.
-			// see: lp:1833763
-			if warningInfo := series.SeriesWarningInfo(charmSeries); len(warningInfo) > 0 {
-				logger.Warningf(
-					"Parsing issues occurred when extracting the distro-info.\n\n"+
-						"  - %s\n"+
-						"\nYou may want to try updating your distro-info using `apt-get update distro-info`\n",
-					strings.Join(warningInfo, "\n  - "),
-				)
-			}
```

Comment:
As we never parse the distro-info for dates any more, we don't need the warning info.